openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -245,8 +245,29 @@ def test_dry_run_gaff_vacuum(benzene_vacuum_system, toluene_vacuum_system,
     )
     unit = list(dag.protocol_units)[0]
 
-    with tmpdir.as_cwd():
-        sampler = unit.run(dry=True)['debug']['sampler']
+    # If we do a lot of GAFF testing, this should be refactored so we don't
```

Comment:
Could you stick an issue number for this here please? That way next person that comes along understands why this all happened.

Reply:
Yes!